### PR TITLE
Update branching docs

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A preview-label label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: Documentation/Branching.md [hunk 1]
```diff
@@ -90,11 +126,13 @@ Holds minimal changes for the purpose of making the upcoming release more stable
    <div>
       <span style="font-weight:bold">Branching Diagram</span>
    </div>
-   <a href="#"><img style="width: 75%" align="center" src="./Images/git-branching-diagram.jpg"></a>
+
+![BranchingDiagram](./Images/BranchingDiagram-DarkMode-v1.1.png#gh-dark-mode-only)
+![BranchingDiagram](./Images/BranchingDiagram-LightMode-v1.1.png#gh-light-mode-only)
```
Comment (KinsonDigitalAdmin) [inline Documentation/Branching.md:131]: @kselena 
I was reviewing the document by rendering it as markdown and I noticed that these images do not exist in the project.
Because they do not exist, they end up as a broken image link in the markdown.  They simply need to be added.

The files can be found in the **Velaptor** project.

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by kselena — COMMENTED** [review]
Approved

**Review by KinsonDigitalAdmin — APPROVED** [review]
